server/rest/src/main/java/org/infinispan/rest/server/operations/mediatypes/printers/ASCIIOutputPrinter.java

```diff
@@ -0,0 +1,19 @@
+package org.infinispan.rest.server.operations.mediatypes.printers;
+
+import java.nio.charset.Charset;
+
+import org.infinispan.CacheSet;
+import org.infinispan.rest.server.operations.mediatypes.OutputPrinter;
+
+public class ASCIIOutputPrinter implements OutputPrinter {
+
+   @Override
+   public byte[] print(String cacheName, CacheSet<?> keys) {
+      return keys.toString().getBytes(Charset.forName("US-ASCII"));
```

Comment:
Hrmm I don't know if we can trust the toString to print the contents of a `CacheSet` in all cases. I would say we should do the iterator or stream ourselves to be 100% sure.